src/Common/src/System/Security/Cryptography/PasswordBasedEncryption.cs

```diff
@@ -418,8 +415,7 @@ private static CryptographicException AlgorithmKdfRequiresChars(string algId)
                         }
                         finally
                         {
-                            CryptographicOperations.ZeroMemory(sourceRent.AsSpan(0, source.Length));
-                            ArrayPool.Return(sourceRent);
+                            CryptoPool.Return(sourceRent, source.Length);
```

Comment:
The stronger version of zeroing isn't needed anymore?  I'm fine with that, just confirming it's intentional.

Reply:
Based on having received feedback on this twice, I've gone ahead and done the \#if in CryptoPool to use CryptographicOperations.ZeroMemory when it's available.